python/src/uagents/protocol.py

```diff
@@ -315,7 +315,7 @@ def manifest(self) -> Dict[str, Any]:
 
         for schema_digest, model in all_models.items():
             manifest["models"].append(
-                {"digest": schema_digest, "schema": model.model_json_schema()}
+                {"digest": schema_digest, "schema": model.schema()}
```

Comment:
Restored the correct function as (I assume) it was mistakingly changed here
https://github.com/fetchai/uAgents/commit/110c1c921da040728d8aa5472b85ce29a6a532f4#diff-6043ffbea937f982ee0f219d8d51acc51c66a5d9ca9faef0a33bb26daf227f81L318
`schema` != `json_schema`
https://docs.pydantic.dev/1.10/usage/models/
https://docs.pydantic.dev/latest/migration/#changes-to-pydanticbasemodel